Brand-Guide.md

```diff
@@ -96,11 +80,11 @@ A clear space also known as an exclusion zone is an area of blank space that sur
 
 1. **Applying Effects**: Avoid adding visual effects such as drop shadows, glows, or bevels to the logo. These effects can distract and compromise the logo's legibility.
 
-1. **Adding Elements**: Do not augment the logo with additional elements or graphics. Adding elements can weaken the overall design and detract from its intended impact.
+1. ~~**Adding Elements**: Do not augment the logo with additional elements or graphics. Adding elements can weaken the overall design and detract from its intended impact.~~
 
 1. **Rotating the Logo**: The logo should be kept in its original orientation and not rotated or displayed at an angle. Changing the orientation can compromise legibility and weaken the logo's impact.
 
-1. **Changing Colors**: Do not alter the logo colors beyond the approved color scheme. Using non-approved colors can compromise visual consistency and weaken the logo's impact.
+1. ~~**Changing Colors**: Do not alter the logo colors beyond the approved color scheme. Using non-approved colors can compromise visual consistency and weaken the logo's impact.~~
```

Comment:
I crossed this off for your consideration not only because the BIDS Connectivity project did this and theirs still looks okay...

https://pestillilab.github.io/bids-connectivity/

... but also because I can imagine a desire for some folks to change colors for inclusivity or to engage in support of one cause or another. Again, these are thoughts and not directives. What do you think?